### ISSUE TITLE
Terms drop down with new -multiple- option

### ISSUE THREAD

**Issue by paaljoachim** [ronalfy/post-type-archive-mapping#20]
Hey

I have a customer who would like a featured custom block post that contains multiple terms.
I would suggest having the last option in the terms drop down the word: -Multiple-

When the user selects Multiple a new text field shows up where they can write in the category names and add commas after each. Or another drop down box shows up showing all the categories with checkboxes next to each. 
Bottom line is having a method to where the user can select multiple categories for 1 custom block.
